test/factory-test.js

```diff
@@ -183,7 +188,7 @@ describe('factory', function() {
           factory.build('person', { age: 30 }, function(err, person) {
             (person instanceof Person).should.be.true;
             person.should.not.have.property('saveCalled');
-            person.name.should.eql('Person 1');
```

Comment:
Actually, I feel that even the factory definitions should not be in beforeEach either. I am open to clean up the tests a bit once this gets merged to avoid the confusion.